packages/@webex/plugin-meetings/test/unit/spec/meeting/index.js

```diff
@@ -3632,6 +3632,82 @@ describe('plugin-meetings', () => {
         });
       });
 
+      describe(`#beRightBack`, () => {
+        const fakeMultistreamRoapMediaConnection = {
+          createSendSlot: sinon.stub().returns({
+            setSourceStateOverride: sinon.stub().resolves(),
+            clearSourceStateOverride: sinon.stub().resolves(),
+          }),
+        };
+
+        beforeEach(() => {
```

Comment:
I think would be good to add also clearing part in  `afterEach`
```
afterEach(() => {
          sinon.restore();
        });
```

Reply:
Also thinking of doing code block 
```
  it('should have #beRightBack', () => {
            assert.exists(meeting.beRightBack);
          });
```

before _describe_ because it's common for two flows, LMKWYT